src/jtag/drivers/ftdi_cjtag.c

```diff
@@ -0,0 +1,965 @@
+/****************************************************************************
+       Module: ftdi_cjtag.c
+     Engineer: Martin Hannon
+  Description: Handles cJTAG comms to the ARC processor.
```

Comment:
Reading that comment I may conclude that this is not of any use for any other architectures, is that correct?